articles/what-are-rich-text-editors.mdx

```diff
@@ -0,0 +1,52 @@
+---
+_schema: default
+_uuid: 7630a593-3a92-49c0-8557-627ddd5112fe
+_created_at: 2024-06-12T16:46:11+12:00
+title: What are rich text editors?
+nav_title: What are rich text editors?
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: Learn about rich text editors in CloudCannon.
+article_category:
+  - Explanation
+tags: []
+related_articles: []
+related_links: []
+explicit_canonical:
+---
+A rich text editor is an interface for editing content on your Site, with tools to format and style your content. Rich text editors use a WYSIWYG editing area, meaning that the way you format the content in the editor will resemble the way it will look on your web page (WYSIWYG is an acronym for What-You-See-Is-What-You-Get).
+
+CloudCannon has three kinds of rich text editors:
+
+* [The Content Editor](/documentation/articles/the-content-editor/)
+* [Editable Regions](/documentation/articles/define-editable-regions-in-your-html/) in the [Visual Editor](/documentation/articles/the-visual-editor/)
+* [Rich Text Inputs](/documentation/articles/rich-text-inputs/)
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/editing/editing-interfaces/2024-04/CloudCannon-Documentation-Content-Editor-Blog.png" alt="A screenshot of the Content Editor, an editing interface for content-heavy markup files." title="The CloudCannon Content Editor" type="screenshot"/>
+
+For each rich text editor, you can configure the WYSIWYG toolbar (including which text and image formatting tools are available for each editor), the upload paths or filenames for any assets uploaded through an editor, and how CloudCannon should handle custom markup.
+
+For more information, please read our documentation on [configuring your rich text editors](/documentation/articles/configure-your-rich-text-editors/).
```

Comment:
```suggestion
```

I don't know that we need this line here since it's somewhat floating, and we link off to the same page in-context elsewhere